src/Widgets/Navigation/NavigationTreeWidget.php

```diff
@@ -2,9 +2,44 @@
 
 namespace Ceres\Widgets\Navigation;
 
+use Ceres\Config\CeresConfig;
 use Ceres\Widgets\Helper\BaseWidget;
+use IO\Helper\Utils;
+use IO\Services\CategoryService;
+use IO\Services\CustomerService;
+use IO\Services\ItemSearch\Factories\Faker\CategoryTreeFaker;
 
 class NavigationTreeWidget extends BaseWidget
 {
     protected $template = "Ceres::Widgets.Navigation.NavigationTreeWidget";
+    
+    protected function getPreviewData($widgetSettings)
+    {
+        /** @var CategoryService $categoryService */
+        $categoryService = pluginApp(CategoryService::class);
+        
+        /** @var CeresConfig $ceresConfig */
+        $ceresConfig = pluginApp(CeresConfig::class);
+        
+        /** @var CustomerService $customerService */
+        $customerService = pluginApp(CustomerService::class);
+        
+        $categories = $categoryService->getNavigationTree(
+            $ceresConfig->header->showCategoryTypes,
+            Utils::getLang(),
+            $ceresConfig->header->menuLevels,
+            $customerService->getContactClassId()
+        );
+        
+        if(!count($categories))
+        {
```

Comment:
PSR-2 Codestyling beachten

Leerzeichen nach if
Leerzeichen nach condition Klammer und dann geschweifte Klammer nicht in eine extra Zeile